### DISCUSSION TITLE
ActionCategory and DesignSituation vs matching Standard+Annex

### DISCUSSION THREAD

**Discussion by damianbobrowski** [Dlubal-Software/RFEM_Python_Client#178] [Q&A] [answered]
Hello

Is there any programmatic way or documentation available to get know **which Action Category** from list of 258 elements:
https://github.com/Dlubal-Software/RFEM_Python_Client/blob/main/RFEM/enums.py#L1851

**is true** for given `current_standard_for_combination_wizard` documented via doc string here:
https://github.com/Dlubal-Software/RFEM_Python_Client/blob/main/RFEM/LoadCasesAndCombinations/loadCasesAndCombinations.py#L21

For example for NATIONAL_ANNEX_AND_EDITION_EN_1990_CEN_2010_04 we have 18 Action Categories matched

![image](https://user-images.githubusercontent.com/26790705/193271333-cd9330a3-f275-47ef-b32b-24b50757efab.png)

**Edit** [03.10.2022]
This question is relevant also for DesignSituationType
https://github.com/Dlubal-Software/RFEM_Python_Client/blob/main/RFEM/enums.py#L1920
![image](https://user-images.githubusercontent.com/26790705/193569449-fc9e3cb3-f2c7-4a2c-9172-83468a46e95c.png)

**Comment by bjoernsteinhagen** [comment, marked as answer]
Hey @damianbobrowski,

Thanks for the post :) We have created an issue for this, it can be found under the Issues Tab, #183 .

Regards,
Björn
